client/app/reader/PdfKeyboardInfo.jsx

```diff
@@ -73,3 +73,19 @@ export const documentsColumns = [{ header: 'Navigate reader',
 { header: 'Shortcut',
   valueName: 'shortcut',
   align: 'left' }];
+
+export const categoryColumns = [{ header: 'Add/Remove categories',
+  valueName: 'categoryInstruction',
+  align: 'left' },
+{ header: 'Shortcut',
+  valueName: 'shortcut',
+  align: 'left' }];
+
+export const categoryInstructions = [
+  { categoryInstruction: 'Add/Remove Medical',
+    shortcut: <span><code><code>alt</code> + <code>shift</code></code> + <code>m</code></span> },
```

Comment:
```suggestion
    shortcut: <span><code>alt</code> + <code>shift</code> + <code>m</code></span> },
```

These ended up with nested `<code>` tags somehow, which isn't consistent with others (see, e.g., `searchInstructions` above

The other two instructions below are the same way.

Reply:
Good catch, updated.